daemon/api.go

```diff
@@ -1168,3 +1177,76 @@ func assertsFindMany(c *Command, r *http.Request) Response {
 func getEvents(c *Command, r *http.Request) Response {
 	return EventResponse(c.d.hub)
 }
+
+type changeInfo struct {
+	Kind    string      `json:"kind"`
+	Summary string      `json:"summary"`
+	Status  string      `json:"status"`
+	Tasks   []*taskInfo `json:"tasks,omitempty"`
+}
+
+type taskInfo struct {
+	Kind    string   `json:"kind"`
+	Summary string   `json:"summary"`
+	Status  string   `json:"status"`
+	Log     []string `json:"log,omitempty"`
+}
+
+func getChanges(c *Command, r *http.Request) Response {
+	query := r.URL.Query()
+	logTail := 1
+	qLogTail := query.Get("log-tail")
```

Comment:
@niemeyer when we discussed this first the idea was to show only the last line but it could be done just at the command level and not in the api